### ISSUE TITLE
Working with @Widgetset("com.vaadin.v7.Vaadin7WidgetSet")

### ISSUE THREAD

**Issue by jonmartinsolaas** [wbadam/autocomplete-extension#15]
I consider using crudui (from the Addon Directory)

`		<dependency>
			<groupId>org.vaadin.crudui</groupId>
			<artifactId>crudui</artifactId>
			<version>2.0.0</version>
		</dependency>`

It uses the above widgetset and v7 compatabilitymode. When enabling the annotation above on my UI autocomplete won't show any list, and I can't see any search performed either on the server side.

Is this problem related to autocomplete-extension?

**Comment by wbadam** [comment]
Hi,
In your case the problem is that `com.vaadin.v7.Vaadin7WidgetSet` doesn't contain the implementation of the extension. You may see a similar error on the console.
`Widgetset 'com.vaadin.v7.Vaadin7WidgetSet' does not contain an implementation for org.vaadin.addons.autocomplete.AutocompleteExtension`

You need to use a widgetset that inherits all the widgetsets you need. When compiling the project with Maven (or executing `vaadin:update-widgetset` goal), your widgetset should be updated accordingly. If you modified your widgetset manually, you may need to inherit the widgetsets yourself.

Bottom line is that you should use your widgetset in your UI rather than the one in the title. If your servlet configuration is right, you won't even need to explicitly set the widgetset in your UI.
